activities/MindMath.activity/js/Game.js

```diff
@@ -95,7 +95,7 @@ var Game = {
         </div>
         <div class="slots-area-footer">
           <button id="btn-restart"
-            v-if="!disabled"
+            v-if="!disabled && mode=='timer'"
```

Comment:
What happens in multi-player mode? I guess it will hide the restart button too so it will forbid to hosting user to restart the game.

Reply:
in multiplayer mode restart button would be in result screen. so it will not have a problem